lib_nbgl/src/nbgl_use_case.c

```diff
@@ -1599,7 +1600,9 @@ void nbgl_useCaseGenericSettings(const char                   *appName,
     pageTitle = appName;
     navType   = GENERIC_NAV;
 
-    memcpy(&genericContext.genericContents, settingContents, sizeof(nbgl_genericContents_t));
+    if (settingContents != NULL) {
+        memcpy(&genericContext.genericContents, settingContents, sizeof(nbgl_genericContents_t));
+    }
```

Comment:
Humf I have indeed not handled this case!
I think I will do it in another PR cause the proper fix is probably different?

Reply:
I've not made 100s of tests but with this fix the top-right icon is OK, and the infos are properly displayed... Maybe there is a simpler one, but it seems difficult...

Reply:
Ok, I was afraid that the info display content won't work, but it does indeed, so this is ok for me!